app/src/interfaces/file/file.vue

```diff
@@ -46,7 +46,11 @@
 
 			<v-list>
 				<template v-if="file">
-					<v-list-item :download="file.filename_download" :href="downloadURL">
```

Comment:
I think there's an oddity with the recent merge from main here. The removal of `downloadURL` which was a recent fix in #15126 results in an unused computed `downloadURL` now. I see that they're functionally the same aside from `downloadURL` not having the `download` query param, but I do personally prefer the use of computed value. @licitdev thoughts on this?

Reply:
I believe that issue #14870 has been fixed alongside #15126. 🤔 

While the `?download` query param isn't necessary to allow for the download in app, it's a nice addition to ensure that the download is forced when users copy the download link. 👍 

I'd personally prefer the use of computed value as well.